apps/dtsse/dtsse-dashboard-github-ingestion/image-policy.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: image.toolkit.fluxcd.io/v1alpha2
+kind: ImagePolicy
```

Comment:
these need referencing

Reply:
sorry, what does that mean?

Reply:
In the kustomization.yaml?

Reply:
I've added them to the automation folder